static/panes/compiler.js

```diff
@@ -925,6 +941,27 @@ Compiler.prototype.onIrViewClosed = function (id) {
     }
 };
 
+Compiler.prototype.onDeviceViewOpened = function (id) {
+    if (this.id === id) {
+        this.deviceButton.prop('disabled', true);
```

Comment:
I think we can consider removing this tracking stuff: we should discuss whether we want to support multiple open device views.